src/sim_explorer/models.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+
+from pydantic import BaseModel
+
+
+class Temporal(Enum):
+    UNDEFINED = 0
```

Comment:
we could change enum base class from `Enum` to `IntEnum`.  This gives some added support for e.g. integer comparison etc.
```suggestion
class Temporal(IntEnum):
```

Reply:
not sure how the removal of the line with `UNDEFINED = 0` made its way in my suggestion for change.
This is a mistake.  @Jorgelmh   better not commit the suggestion directly, but change it (if you want) manually in this case.. :-)